cmd/trivy/main.go

```diff
@@ -117,6 +117,11 @@ OPTIONS:
 			Value: utils.DefaultCacheDir(),
 			Usage: "use as cache directory, but image cache is stored in /path/to/cache/fanal",
 		},
+		cli.StringFlag{
+			Name:  "ignorefile",
+			Value: ".trivyignore",
```

Comment:
Then, specify the default value
```suggestion
			Value: vulnerability.DefaultIgnoreFile,
```